inst/htmljs/ldavis.js

```diff
@@ -1,36 +1,42 @@
-LDAvis = function(to_select, json_file) {
+/* Original code taken from https://github.com/cpsievert/LDAvis */
+/* Copyright 2013, AT&T Intellectual Property */
+/* MIT Licence */
+
+'use strict';
+
```

Comment:
@bmabey I'll admit, I didn't know about "use strict" until now, but it seems like a good idea!

@kshirley In case you're curious, I thought this post does a great job of explaining it -> http://ejohn.org/blog/ecmascript-5-strict-mode-json-and-more/